docker/build_and_push.Dockerfile

```diff
@@ -96,4 +96,4 @@ WORKDIR /app
 ENV LANGFLOW_HOST=0.0.0.0
 ENV LANGFLOW_PORT=7860
 
-ENTRYPOINT ["python", "-m", "langflow", "run"]
\ No newline at end of file
+CMD ["python", "-m", "langflow", "run"]
```

Comment:
If this PR changes the entrypoint to cmd, we should consider the proper use of entrypoint and cmd. ```ENTRYPOINT sets the process to run, while CMD supplies default arguments to that process.```
So it makes sense to still keep the existing entrypoint but allow user to override their command line options in this line. I hope this makes sense.
It's described by this post https://spacelift.io/blog/docker-entrypoint-vs-cmd#example-using-entrypoint-and-cmd-together as

Reply:
I think I understand your point about technical rigor. But do you see this change as a practical issue? I would like to simplify things